docs/gemini/gemini.rst

```diff
@@ -132,6 +132,34 @@ terms need be passed into the method.
                           0.0           Full Frame                         -- Gemini-North    True ...
 
 
+Authenticated Sessions
+----------------------
+
+The Gemini module allows for authenticated sessions using your GOA account.  This is the same account you login
+with on the GOA homepage at `https://archive.gemini.edu/`_.  The `login()` method returns `True` if successful.
```

Comment:
fix to the docs build
this should fix the failing docs build
```suggestion
with on the GOA homepage at `<https://archive.gemini.edu/>`__.  The `astroquery.gemini.ObservationsClass.login` method returns `True` if successful.
```

Reply:
ok done